decidim-core/app/views/layouts/decidim/_offline_banner.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="flash callout primary small js-offline-message" style="display: none">
+  <%= t(".cache_version_page") %>
+</div>
+<script>
+  if (!navigator.onLine) {
+    document.querySelector(".js-offline-message").style.display = "block"
```

Comment:
I'd add some logic to don't display this banner when you are showing the offline page, it's kind of confusing for users:

<img width="893" alt="Screenshot 2022-01-15 at 07 52 17" src="https://user-images.githubusercontent.com/17616/149612504-7f8aeb1e-86ca-4985-9c98-b13cbfecb699.png">

Reply:
Done